frontends/concrete-python/concrete/fhe/mlir/context.py

```diff
@@ -116,6 +117,20 @@ def typeof(self, value: Union[ValueDescription, Node]) -> ConversionType:
         assert isinstance(value.dtype, Integer)
         bit_width = value.dtype.bit_width
 
+        # TODO: what about the element type? only unsigned? or not eint at all?
```

Comment:
I guess it depends on how it's done in TFHE-rs. We should think about which operations would be performed on this type and use eint or esint accordingly. We should also test correct transfer of sign (if input was -3 in TFHE-rs, it should correctly be converted to -3 in Concrete).